firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -223,12 +223,20 @@ private final void doRegistration() {
       PersistedInstallationEntry persistedInstallationEntry =
           persistedInstallation.readPersistedInstallationEntryValue();
 
+      // Default value of iidToken
+      String iidToken = null;
+
       // New FID needs to be created
       if (persistedInstallationEntry.isNotGenerated()) {
 
         // For a default firebase installation read the existing iid. For other custom firebase
         // installations create a new fid
         String fid = readExistingIidOrCreateFid();
+
+        // For a default firebase installation read the stored star scoped iid token. This token
+        // will be used for authenticating the iid on FIS server.
+        iidToken = iidStore.readToken();
```

Comment:
Please call this instanceIdMigrationAuth (or something similar) and only describe in the comment that this is an IID-Token.
The reason is that semantically this is the authentication secret for the migration step and that is what is important for us.

Reply:
Done. PTAL.